### PR TITLE
Improve documentation and clarify behavior of test case

### PR DESCRIPTION
**Pull request by rjollos**

### CODE REVIEW

**Comment by rjollos** [comment]
@neverpanic @aaugustin - requesting review of changes. Thanks!

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2012-2014, Aymeric Augustin.
+Copyright (c) 2012-2016, Aymeric Augustin.
```
Comment (aaugustin) [inline LICENSE]: and contributors.

File: runtests.py [hunk 2]
```diff
@@ -13,6 +13,14 @@
...
 import shutil
 import signal
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2012-2016 Aymeric Augustin <aymeric.augustin@m4x.org>
+# All rights reserved.
+#
+# This software is licensed as described in the file LICENSE, which
+# you should have received as part of this distribution.
+
```
Comment (aaugustin) [inline runtests.py]: If you want to add this add it at the top!

Comment (rjollos) [inline runtests.py]: Thanks for spotting, that was a copy and paste error.

File: LICENSE (hunk 1 above)
Comment (rjollos) [inline LICENSE]: I'll remove the copyright lines before committing. Thanks.

**Comment by rjollos** [comment]
Sorry for the messy history. I intended to squash 373e1ab9bfcf128d6f51b3b28a95694b04b55adb before commit, but ended up pushing the wrong branch.

**Pull request merged**